app/assets/stylesheets/mobile/login-signup-page.scss

```diff
@@ -1,8 +1,24 @@
 // Shared styles
 
+.login-page,
+.signup-page,
+.invite-page {
+  #main-outlet,
+  #main-outlet-wrapper {
+    padding: 0;
+  }
+}
+
 .login-fullpage,
 .signup-fullpage,
 .invites-show {
+  justify-content: flex-start;
+
+  html.keyboard-visible:not(.ios-device) & {
```

Comment:
Are you sure we don't need this on iOS?

Reply:
Yes, I tested it, and it only affects Android.